docs/JK.Common/Patterns/ServiceLocator/IServiceLocator.md

```
@@ -0,0 +1,37 @@
+[Docs](../../../README.md) > [JK.Common](../../README.md) > IServiceLocator
+
+# IServiceLocator
+
+**Namespace:** `JK.Common.Patterns.ServiceLocator`
+
+Service Locator design pattern interface.
+
+### Locate`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)